rustls/src/client/handy.rs

```diff
@@ -182,14 +179,11 @@ impl client::ResolvesClientCert for FailResolveClientCert {
 pub(super) struct AlwaysResolvesClientCert(Arc<sign::CertifiedKey>);
 
 impl AlwaysResolvesClientCert {
-    #[cfg(feature = "ring")]
     pub(super) fn new(
+        priv_key: Arc<dyn sign::SigningKey>,
```

Comment:
Nit: I would probably spell out `priv_key` (into `private_key`), perhaps because I feel like the abbreviation isn't as widely used as it is for `pub_key`. Might just be a personal opinion, feel free to ignore.